org.winehq.Wine.DLLs.dxvk.yml

```diff
@@ -0,0 +1,98 @@
+id: org.winehq.Wine.DLLs.dxvk
+build-extension: true
+runtime: org.winehq.Wine
+runtime-version: stable-21.08
+sdk: org.freedesktop.Sdk//21.08
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.mingw-w64
+build-options:
+  prefix: /app/dlls/dxvk
+  prepend-path: /app/dlls/dxvk/bin
+  append-path: /usr/lib/sdk/mingw-w64/bin
+  #FIXME -fstack-clash-protection causes ICE
+  # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=90458
+  cflags: -O2 -g -pipe
```

Comment:
Would passing  `-fno-stack-clash-protection` be sufficient?

Reply:
Looks like it, thanks for the hint.